app/code/community/Bolt/Boltpay/Model/ShippingAndTax.php

```diff
@@ -222,13 +236,16 @@ public function getShippingAndTaxEstimate( Mage_Sales_Model_Quote $quote, $boltO
                     $metaData = array('quote' => var_export($quote->debug(), true));
                     $this->boltHelper()->logWarning($exception->getMessage(),$metaData);
                     $this->boltHelper()->notifyException($exception, $metaData);
+                    benchmark("Creating New Estimate: Logged shipping rate error for {$rate->getCarrierTitle()}");
                     continue;
                 }
 
                 if ($parentQuote) $quote->setCouponCode($parentQuote->getCouponCode());
 
                 $quote->setShouldSkipThisShippingMethod(false);
-                $this->applyShippingRate($quote, $rate->getCode());
+                benchmark("Creating New Estimate: Applying shipping rate {$rate->getCode()}");
+                $this->applyShippingRate($quote, $rate->getCode(), false);
```

Comment:
@danac-gs 
In the `for loop`, when I collect totals, I do not recollect shipping rates on the address.  That is what the false parameter is saying.  I only change the shipping method and call collect-totals.